app/assets/stylesheets/application.css.scss

```diff
@@ -518,4 +518,19 @@ span.twitter-typeahead {
 
 .paper-preview-not-available {
   text-align: center;
+}
+
+.latest-run {
+  font-size: 0;
+  line-height: 0;
+  height: 14px;
+  width: 14px;
+  display: inline-block;
+  vertical-align: middle;
+  border: 7px solid #9b0000;
+  border-radius: 100%;
+}
+
+.latest-run--ok {
+  border-color: #5cb85c;
```

Comment:
Color literals like `#5cb85c` should only be used in variable declarations; they should be referred to via variable everywhere else.